### PR TITLE
Add note about precompiled BPL

### PR DESCRIPTION
**Pull request by denieu**
Adds note about possible problems in precompiled BPLs, as reported in the issue #51.

I'm not sure which version the Delphi 12 Athens BPL is compiled in, but I'll adjust it after the review if necessary.

### CODE REVIEW

**Comment by fourls** [comment]
Hi @denieu,

I'm wondering if it might be better to add a message like this at the bottom of each release, since it may change between releases. I could do that fairly easily without needing a code change.

I guess we could still include a brief disclaimer in the README, just saying something like "Please note that there may be some compatibility issues when installing the precompiled packages on a different version than they were compiled for."

What do you think?

**Comment by denieu** [comment]
I think it's great, this way we prevent the README from becoming outdated when the compiler used changes.

File: README.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ Installation steps:
 2. Unzip the folder and run the included `install.bat` script.
 3. Done!
 
+The precompiled BPLs available in [the latest release](https://github.com/integrated-application-development/delphilint/releases/latest) were compiled in specific versions of Delphi, using them in different versions from which they were compiled may cause compatibility problems, in this case to continue [build from source](#building-from-source).
```
Comment (fourls) [inline README.md:47]:
```suggestion
> [!NOTE]
> Using the precompiled BPLs in different minor versions from which they were compiled can cause compatibility problems in some circumstances. If you encounter issues, please [build from source](#building-from-source).
```

**Review by fourls — CHANGES_REQUESTED** [review]
Looks good for the most part - I've just made a couple of suggestions to make it more concise and stand out more on the page.

**Review by fourls — APPROVED** [review]
Thanks @denieu, looks good.

I'll update the releases soon.

**Pull request merged**